### DISCUSSION TITLE
models/272/DPA-2.2.0-v3.0.0b3

### DISCUSSION THREAD

**Discussion by giscus** [deepmodeling/AIS-Square#235] [Announcements]
# models/272/DPA-2.2.0-v3.0.0b3

https://www.aissquare.com/models/detail?pageType=models&name=DPA-2.2.0-v3.0.0b3&id=272

**Comment by scott-5** [comment]
In `dp --pt change-bias`, -m option may be --model-branch.

**Reply by scott-5** [reply]
There is another question that:
In [https://bohrium.dp.tech/notebooks/57552161357], zero-shot is done by `dp --pt --finetune`, but here is `dp --pt change-bias`, what are differences between them?

**Reply by 1azyking** [reply]
Practically, to execute zero-shot learning, `dp --pt train --finetune` with `num_steps` of 0 is identical to `dp --pt change-bias`.

**Comment by QuantumMisaka** [comment]
Is there an online installation method for deepmd-3b3 ?

**Reply by 1azyking** [reply]
Sure, please refer the [official page](https://docs.deepmodeling.com/projects/deepmd/en/v3.0.0b3/install/easy-install.html) of installation. Practically, you can try the following command to install the deepmd-kit v3.0.0b3:
```bash
pip install git+https://github.com/deepmodeling/deepmd-kit@v3.0.0b3
```
